spec/rubocop/cop/rails/transaction_exit_statement_spec.rb

```diff
@@ -0,0 +1,29 @@
+# frozen_string_literal: true
+
+RSpec.describe RuboCop::Cop::Rails::TransactionExitStatement, :config do
+  it 'registers an offense when exit statements are detected in transactions' do
+    expect_offense(<<~RUBY)
+      ApplicationRecord.transaction do
+        return if user.active?
+        ^^^^^^ Exit statements such as `return`, `break` and `throw` are not allowed.
+
+        break if user.active?
+        ^^^^^ Exit statements such as `return`, `break` and `throw` are not allowed.
+
+        throw if user.active?
+        ^^^^^ Exit statements such as `return`, `break` and `throw` are not allowed.
+      end
+
+      ApplicationRecord.transaction { return if user.active? }
+                                      ^^^^^^ Exit statements such as `return`, `break` and `throw` are not allowed.
```

Comment:
Ditto.

Reply:
I'm unsure if testing single-line blocks are necessary, I have removed this test.